### ISSUE TITLE
Unstable formatting for enum w/ comment

### ISSUE THREAD

**Issue by jellopuddingstick** [jhipster/prettier-java#454]
**Prettier-Java 1.0.1**

The following code has to be formatted twice in order to "stabilize".

**Input:**

```java
public enum Bug {
  FOO,
  BAR, // comment
  ;

  private int x;
}
```

**First Output:**

```java
public enum Bug {
  FOO,
  BAR;// comment

  private int x;
}
```

**Second Output:**

```java
public enum Bug {
  FOO,
  BAR; // comment

  private int x;
}
```

**Expected behavior:**

The code should only need to be formatted once to produce the "stable" output.

**Comment by pascalgrimaud** [comment]
Adding a bounty as it's a bug

**Comment by somayaj** [comment]
this is an issue with the prettier-maven-plugin 0.8 version. I ran through your code and it was unstable with the version that you specified, the plugin-write task had to executed twice to get it work correctly wth the comment str as you mention in the defect. 
But, I tried the same code with 0.15 version and it worked correctly the first time around. I would suggest as a fix to upgrade to 0.15 and retest.

**Comment by somayaj** [comment]
Hi, was there any update on this?

**Comment by pascalgrimaud** [comment]
@somayaj : I don't understand your question. Is this ticket need an update ? According to your comment, it seems to be fixed, so for me, we can close this ticket

Don't hesitate to tell me if I'm wrong or if I didn't understand something :-)

**Comment by somayaj** [comment]
Great just upgrade the version of the maven plugin and you should be good,
do I get the 100$ ? Curious

**Comment by pascalgrimaud** [comment]
Sorry, but to be fair, the bounty should go to the one who did the fix in prettier-java between 0.8  and 0.15 :-)

ping @clementdessoude maybe you know who fixed this issue ?

@somayaj : but don't hesitate to participate to this project or another one of JHipster Organization. We'll be glad to help you. If there are some efforts to fix issues, I can add bounty !

**Comment by jellopuddingstick** [comment]
Hey all, not sure why this was marked closed; it is not fixed as of `prettier@2.5.0` and `prettier-plugin-java@1.6.0`.

> this is an issue with the prettier-maven-plugin 0.8 version

I'm not using the prettier-maven-plugin, I'm invoking prettier directly from the command line. Full, minimal reproduction steps:

```bash
tee package.json << 'EOF'
{
  "name": "jhipster/prettier-java#454"
}
EOF

npm install --save-dev prettier prettier-plugin-java

tee Bug.java << 'EOF'
public enum Bug {
  FOO,
  BAR, // comment
  ;

  private int x;
}
EOF

npx prettier --write . # Produces "First Output", above
npx prettier --check . # Fails

npx prettier --write . # Produced "Second Output", above
npx prettier --check . # Succeeds
```
